dockstore-client/src/main/java/io/dockstore/client/cli/nested/CromwellLauncher.java

```diff
@@ -49,6 +52,10 @@ public CromwellLauncher(AbstractEntryClient abstractEntryClient, DescriptorLangu
     public void initialize() {
         // initialize cromwell location from ~/.dockstore/config
         INIConfiguration config = Utilities.parseConfig(abstractEntryClient.getConfigFile());
+        cromwellVmOptions = (List)(config.getList("cromwell-vm-options"));
```

Comment:
Is this cast necessary? 
https://commons.apache.org/proper/commons-configuration/apidocs/org/apache/commons/configuration2/AbstractConfiguration.html#getList-java.lang.Class-java.lang.String-

Reply:
Yes otherwise there is an incompatible types error: List<java.lang.string> vs List<java.lang.object>